docs/learn.rst

```diff
@@ -260,6 +261,15 @@ To read digital books in Kolibri you have several available controls inside the
 
 .. _lessons:
 
+Slideshow Viewer options
```

Comment:
You would have to move it up a bit because of the `lessons` label declaration now pointing to the slideshow header.

Reply:
As @benjaoming noted, either move the `.. lessons:` reference so it immediately precedes the `Lessons` heading, or move the whole slideshow section above `.. lessons:`.

Reply:
Haha, @radinamatic - I'm LOL'ing out loud right now :)